CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.21.4]
+---------
+* feat: Updates messaging on disable expiry for learner credit
```

Comment:
nit: if this a `feat`, then the minor version should be bumped, not the patch version (i.e., `4.22.0` vs. `4.21.4`). Given this is updating help text, seems like it'd be more in line with `fix` (i.e., not really introducing net-new functionality).